src/_locales/en_US/messages.json

```diff
@@ -359,6 +359,22 @@
         "message": "Privacy Policy",
         "description": "Shown at the bottom of the intro page."
     },
+    "intro_pin_nudge": {
+        "message": "$START_SPAN_TAG$First, pin Privacy Badger to your toolbar$END_SPAN_TAG$ for easy access to the \"$DISABLE_FOR_SITE$\" button.",
+        "description": "Part of a Chrome-only overlay on the intro page",
+        "placeholders": {
+            "start_span_tag": {
+                "content": "<span style='font-family:\"OpenSans-Bold\",sans-serif;'>"
```

Comment:
Let's use the `pin-cta` ID or some other CSS identifier instead. Shorter and less likely for us to have to edit locales when making changes in the future.

Reply:
good point!